docs/workflows/phylogenetic_construction/augur.md

```diff
@@ -290,6 +290,7 @@ The Nextstrain team hosts documentation surrounding the Augur workflow → Auspi
 | auspice_input_json | File | JSON file used as input to Auspice |
 | combined_assemblies | File | Concatenated FASTA file containing all samples |
 | distance_tree | File | The distance tree created in Newick (.nwk) format |
+| iqtree_model_used | String | The iqtree model used during augur tree |
```

Comment:
can you provide that link to the iqtree documentation page that lists the models available?

probalby in the inputs, not the outptus

Reply:
http://www.iqtree.org/doc/Substitution-Models

Reply:
awesome! can you add that to the docs, in the inputs?

Reply:
Done. Let me know if you like the formatting.

Reply:
perf, thanks